### PR TITLE
fix(file.js, pwa.js, package.json): fix Windows backslash paths being…

### LINKED ISSUE #36: Incorrect file paths generated on Windows
**Describe the bug**
On Windows, the asset paths generated have double back-slashes (sometimes three) at the first directory level. This causes cli.test.js to fail, also user has to manually correct the asset paths generated.

**To Reproduce**
1. Execute cli command `git clone https://github.com/onderceylan/pwa-asset-generator.git`
2. `cd` to project root and execute cli command: `npm run jest`
3. See error

**Expected behavior**
cli.test.js should pass

**System (please complete the following information):**
 - OS: windows 10
 - node version: v10.15.3
 - npm version: 6.11.2
 - cli version: 1.1.6

**Screenshots**
image

### PR DESCRIPTION
**Pull request by leonelngande**
… added to icon and html content

Introduce the slash npm package to use in converting backslash paths generated for saved images to
slash paths, which are needed for generated html and manifest json content

fix #36

### CODE REVIEW

File: helpers/file.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const fs = require('fs');
 const path = require('path');
+const slash = require('slash');
```
Comment (onderceylan) [inline helpers/file.js]: This package looks like a perfect fit!

Comment (leonelngande) [reply helpers/file.js]: Yep, it just makes the double slash issue go away ;)

File: helpers/pwa.js [hunk 2]
```diff
@@ -9,8 +9,10 @@ const generateIconsContentForManifest = (savedImages, manifestJsonPath) => {
     )
     .map(({ path, width, height }) => ({
       src: manifestJsonPath
-        ? file.getRelativeFilePath(manifestJsonPath, path)
-        : path,
+        ? file.convertBackslashPathToSlashPath(
```
Comment (onderceylan) [inline helpers/pwa.js]: I think it's time for a refactor here as this has been more complicated to follow as part of ternary expression and feels like we start to repeat some code. I propose introducing a generic function in file.js service, such as;

```javascript
export const getRelativeImagePath = (outputFilePath, imagePath) => {
    if (outputFilePath) {
        return convertBackslashPathToSlashPath(getRelativeFilePath(outputFilePath, imagePath));
    }
    return convertBackslashPathToSlashPath(imagePath);
}
```

Then we can simply call this function in 3 places where we need this functionality.

**Comment by onderceylan** [comment]
Looks like your branch is out-of-date with the base branch. Are you able to update branch here?

File: helpers/pwa.js (hunk 2 above)
Comment (leonelngande) [reply helpers/pwa.js]: Okay makes sense. Should I do this refactor and push, or you can make the change directly?

Comment (onderceylan) [reply helpers/pwa.js]: Please do so, it's your PR :)

Comment (leonelngande) [reply helpers/pwa.js]: Ok on it!

**Pull request merged**